src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -599,15 +599,23 @@ def _update_lock_parameters(parameters, level, notes, lock_id, lock_type):
     if lock_type is not None:
         parameters.type = lock_type
 
-def update_lock(name, resource_group_name=None,
-                level=None, notes=None, lock_id=None, lock_type=None):
+def update_lock(name, resource_group_name=None, level=None, notes=None):
+    '''
+    :param name: The name of the lock
```

Comment:
Please move this docstring help into the _help.py files. The reason is because the live doc's Edit button points directly to that file, making it easy for someone to edit the help in the future.